packages/example/next.config.js

```diff
@@ -1,4 +1,19 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
+  i18n: {
+    defaultLocale: 'en_US',
+    locales: [
+      'en_US',
+      'es_419',
+      'fr_FR',
+      'ja_JP',
+      'pt_BR',
+      'zh_CN',
+      'id_ID',
+      'hi_IN',
+      'tr_TR',
+      'ru_RU',
+    ],
+  },
   reactStrictMode: true,
```

Comment:
These will be the `locales` we will support. People who use this feature will also have access to routing which will help `rainbowkit` stay in sync with any changes to the `locale` prop.

Example from next.js:

![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/9a8263fd-dde7-4d4c-bea2-ea1a4ccdf916)